.goreleaser.yml

```
@@ -89,4 +83,13 @@ docker_manifests:
     - "{{ .Env.APISERVER_IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-ppc64le"
     - "{{ .Env.APISERVER_IMAGE_REPO }}:{{ .Env.IMAGE_TAG }}-s390x"
 release:
-  disable: '{{ ne .Env.ENABLE_RELEASE_PIPELINE "true" }}'
\ No newline at end of file
+  disable: '{{ ne .Env.ENABLE_RELEASE_PIPELINE "true" }}'
+  extra_files:
+  - glob: 'catalogd.yaml'
+  header: |
+    ## Installation
+    ```bash
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/{{ .Env.CERT_MGR_VERSION }}/cert-manager.yaml
+    kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
+    kubectl apply -f https://github.com/operator-framework/operator-controller/releases/download/{{ .Tag }}/operator-controller.yaml
```

```suggestion
    ## Installation
    ```bash
    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/{{ .Env.CERT_MGR_VERSION }}/cert-manager.yaml
    kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
    kubectl apply -f https://github.com/operator-framework/catalogd/releases/download/{{ .Tag }}/catalogd.yaml
```

---

Ah! :facepalm:

---

resolved in https://github.com/operator-framework/catalogd/pull/25/commits/392ab9912488a9da4caa9932d37a723ece890973